templates/core/core-secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $secret := include "harbor.core" . }}
```

Comment:
As you define the name here, shouldn't you use it wherever possible? (`metadata:` -> `name:` here and I assume in all other secrets...

Reply:
Done